### PR TITLE
remove extra lines that were not required

### PR DESCRIPTION
**Pull request by JAMSADIQ**
fixing some extra code

### CODE REVIEW

File: popde/density_estimate.py [hunk 1]
```diff
@@ -67,9 +67,6 @@ def __init__(self, data, weights=None, input_transf=None, stdize=False,
 
         self.data = np.asarray(data)
         self.ndim = self.data.shape[1]
-        self.weights = np.atleast_1d(weights).astype(float)
```
Comment (tdent) [inline popde/density_estimate.py:70]: why don't we want to keep these lines but only perform them if weights is not None? 

I.e. 
```
if weights is not None:
    self.weights = (etc...)
    if len(self.weights) != self.data.shape[0]"
        (etc)
else:
    self.weights = None
```

Comment (JAMSADIQ) [reply popde/density_estimate.py:70]: Well, if weights are None than these lines will raise error. These checks are there in next few lines of the code so this is redundant

Comment (tdent) [reply popde/density_estimate.py:70]: I don't know what you mean Jam. I said perform the checks etc if weights is not None. If weights is a valid option for this class then we should perform the checks here and remove the ones done in a subclass.

Comment (JAMSADIQ) [reply popde/density_estimate.py:70]: If you check the next few lines of the code there is this check done. https://github.com/tdent/pop-de/blob/main/popde/density_estimate.py#L82

**Review by tdent — APPROVED** [review]

**Comment by tdent** [comment]
Oh. I did not see the other lines just afterwards because github does not display those lines when I am looking at the changes in this PR.

**Pull request merged**